### PR TITLE
Added KademliaConstants class.

### PR DESCRIPTION
**Pull request by ricdezen**
Added KademliaConstants class containing some often used constants.

### CODE REVIEW

File: kademlia/src/main/java/ingsw/group1/kademlia/KademliaConstants.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package ingsw.group1.kademlia;
+
+/**
+ * Class defining constants to be used in a distributed network using the Kademlia protocol.
+ */
+public class KademliaConstants {
```
Comment (RicDeZen) [inline kademlia/src/main/java/ingsw/group1/kademlia/KademliaConstants.java]: Codacy Issue found: [The utility class name 'KademliaConstants' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4743321)

**Comment by RicDeZen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4743321&bid=15829188)

**Review by turcato-niccolo — APPROVED** [review]

File: kademlia/src/main/java/ingsw/group1/kademlia/KademliaConstants.java (hunk 1 above)
Comment (ricdezen) [inline kademlia/src/main/java/ingsw/group1/kademlia/KademliaConstants.java:6]: Codacy Issue found: [The utility class name 'KademliaConstants' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4743321)

**Comment by ricdezen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4743321&bid=15829188)

**Pull request merged**